WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -132,4 +132,10 @@ def alert(type, content=nil, note: false, &block)
     content.prepend content_tag(:strong, "Note: ") if note
     content_tag :div, content, class: "alert alert-#{type}"
   end
+
+  def users_to_sentence(users, include_email: false)
+    users.sort_by(&:name).map do |user|
+      include_email ? mail_to(user.email, user.name) : user.name
+    end.to_sentence(last_word_connector: " and ").html_safe
```

Comment:
What's the point of this `last_word_connector`? Are you trying to avoid the [serial comma](https://en.wikipedia.org/wiki/Serial_comma)?

```ruby
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> bin/rails console
Loading development environment (Rails 4.2.1)
irb(main):001:0> %w(foo bar baz).to_sentence
=> "foo, bar, and baz"
irb(main):002:0> %w(foo bar baz).to_sentence(last_word_connector: " and ")
=> "foo, bar and baz"
irb(main):003:0> 
```

We use regular `to_sentence` in a lot of places, so I don't think this is the right place to go about changing that.